### DISCUSSION TITLE
To add SSH or not?

### DISCUSSION THREAD

**Discussion by Volte6** [GoMudEngine/GoMud#84] [General]
I've seen some muds add support for SSH connections. Not sure it's worth it, since to me MUD's have always been telnet, but SSH is a secure option (for what that's worth in a mud), and if other MUDs are supporting it, maybe it's in demand.

Is it worth putting in the effort to support a third protocol? Telnet/Web/SSH?

**Comment by ArmoredArtist** [comment]
Telnet has been standard for MUDs for years and the risks in such an environment are generally low I think. It depends on the effort it would take (Honestly, that is far beyond my skill set to judge)
